File: 3-10-2022/3.R
numbers = c(10, 20, 30, 40, 50, 60, 70, 80, 90, 100)

print(numbers[c(2, 4, 6)])
print(numbers[c(1, 5, 6, 9)])
print(numbers[c(9, 10, 2, 3)])
print(numbers[c(2, 5, 3, 6)])
print(numbers[-3])
print(numbers[c(-1, -5)])

print(numbers[c(-1, -3, -5, -7, -8, -9, -10)])
print(numbers[c(-2, -3, -4, -7, -8, -10)])
print(numbers[c(-1, -4, -5, -6, -7, -8, -9)])
print(numbers[c(-1, -4, -7, -8, -9, -10)])

range(c(1, 8))

print(numbers[2:6])
print(numbers[6:10])
print(numbers[5:8])

for (index in 1:length(numbers)) {
  print(paste0(numbers[index]))
}

File: 7-10-2022/3.py
import pandas as pd

s1 = pd.Series([n * 10 for n in range(1, 1 + 10)])

s2 = pd.Series(tuple([n * 10 for n in range(1, 1 + 10)]))
print(s2)

index_ = ['model', 'company', 'price']
values = ['iphone', 'Apple', 140000]
s3 = pd.Series(values, index=index_)
print(s3)

s4 = pd.Series([23, 45, 29, 30, 10, 29, 19, 38])

# positive indexing

# negative indexing
# print(s4[-2])

# slicing
print(s4[-2:-8:-2])

# filtering
print(s4[s4 > 20])
print(s4[s4 < 23])
print(s4[s4 % 2 == 0])

